interface/wc_treasury.gfx

```diff
@@ -87,6 +87,14 @@ spriteTypes = {
 		name = "GFX_inv_misc_gem_amethyst_02_small"
 		texturefile = "gfx\\interface\\inventory\\artifacts\\inv_misc_gem_amethyst_02_small.dds"
 	}
+	spriteType = {
+		name = "GFX_inv_misc_book_08"
+		texturefile = "gfx\\interface\\inventory\\artifacts\\inv_misc_book_08_small.dds"
+	}
+	spriteType = {
+		name = "GFX_inv_misc_book_08_small"
+		texturefile = "gfx\\interface\\inventory\\artifacts\\inv_misc_book_08_small.dds"
+	}
```

Comment:
Put new lines in the end of file, otherwise there will be conflicts on other branches